### PR TITLE
#165879553: Fix CORS policy errors on admin user page

### PR DESCRIPTION
**Pull request by emasys**
#### What does this PR do?
Fixes a CORS bug associated with a bad request to the server
#### Description of Task to be completed?
Fix the bug and show a meaningful and simple error message that the users can easily grab or just ensure the error does not show.
#### How should this be manually tested?
- pull this branch `bg-fix-cors-error-165879553`
- log in as an admin
- click on `users` link on the side navigation bar 
#### What are the relevant pivotal tracker stories?
- [#165879553](https://www.pivotaltracker.com/story/show/165879553)
#### Background Context
The loader indicator only appears when there is an actual network request. 
#### Screenshots (If Applicable)
if the list is empty
Screenshot 2019-05-08 at 2 30 12 PM
Screenshot 2019-05-08 at 2 36 44 PM

### CODE REVIEW

**Comment by Maxcutex** [comment]
@emasys  pls request review from @ayz4sci or @SoniaRMK since they are more into the project than me or @sulenchy

**Comment by Maxcutex** [comment]
Also try and make sure the test is passing

**Comment by sulenchy** [comment]
@emasys You can add screenshots(before and after) of the browser's console.

File: client/src/components/Admin/Users/Users.jsx [hunk 1]
```diff
@@ -1,66 +1,81 @@
...
 import Select from 'react-select';
+import Loader from '../../common/Loader/Loader';
 
 // Actions
 import { createAdminUser, getAllAdminUsers } from '../../../actions/admin/adminUserAction';
+import EmptyContent from "../../common/EmptyContent";
 
 
+/**
+ *
```
Comment (sulenchy) [inline client/src/components/Admin/Users/Users.jsx]: You can remove one or two of these asterisks.

**Pull request merged**